src/ui/components/KeyboardShortcuts.tsx

```diff
@@ -15,24 +15,33 @@ function setupShortcuts() {
 
 const globalShortcuts = setupShortcuts();
 
-function KeyboardShortcuts({ viewMode, setSelectedPrimaryPanel, setViewMode }: PropsFromRedux) {
+function KeyboardShortcuts({
+  viewMode,
+  setSelectedPrimaryPanel,
+  togglePaneCollapse,
+  setViewMode,
+}: PropsFromRedux) {
   const openFullTextSearch = () => {
     if (viewMode !== "dev") {
       setViewMode("dev");
     }
-
     setSelectedPrimaryPanel("search");
   };
+  const toggleLeftSidebar = () => {
+    togglePaneCollapse();
+  };
 
   // The shortcuts have to be reassigned every time the dependencies change,
   // otherwise we end up with a stale prop.
   useEffect(() => {
     if (!globalShortcuts) return;
 
     globalShortcuts.on("CmdOrCtrl+Shift+F", openFullTextSearch);
+    globalShortcuts.on("CmdOrCtrl+B", toggleLeftSidebar);
 
     return () => {
       globalShortcuts.off("CmdOrCtrl+Shift+F", openFullTextSearch);
+      globalShortcuts.off("CmdOrCtrl+B", toggleLeftSidebar);
```

Comment:
I like it. And think we should do it, but it's funny if you search for this keyboard shortcut you'll find references for where it used to toggle a breakpoint on a line